src/crepe/system/RenderSystem.h

```diff
@@ -49,7 +49,8 @@ class RenderSystem : public System {
 	 *  constructor is now protected i cannot make tmp inside
 	 * \return true if particles have been rendered
 	 */
-	bool render_particle(const Sprite & sprite, const double & scale);
+	bool render_particle(const Sprite & sprite, const float & transform_angle,
+						 const float & scale);
```

Comment:
Is er een reden dat je hier niet gewoon de `const Transform &` mee geeft?

Reply:
niet alle data van de transform is nodig.